### PR TITLE
feat: new Zinnia API `stationId`

### PR DESCRIPTION
**Pull request by PatrickNercessian**
DRAFT: DO NOT MERGE

Links:
filecoin-station/roadmap#96

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /target
+*.DS_Store
```
Comment (juliangruber) [inline .gitignore:2]:
```suggestion
*.DS_Store

```

File: daemon/main.rs [hunk 2]
```diff
@@ -178,6 +179,7 @@ mod tests {
             cache_root: temp.join("cache").to_string_lossy().into(),
             state_root: temp.join("state").to_string_lossy().into(),
             wallet_address: "f1test".to_string(),
+            station_id: "zinnia-dev".to_string(),
```
Comment (juliangruber) [inline daemon/main.rs:182]: I think that only passing through the environment variable "station_id" isn't ideal, as Zinnia ideally doesn't know anything about Station. Instead there should be a way to pass any key value pair into the module context. I believe that implementing this is out of scope for this PR, so just noting here that I would like to improve this in the future.

File: daemon/args.rs [hunk 3]
```diff
@@ -9,6 +9,10 @@ pub struct CliArgs {
     #[arg(long, short = 'w', env = "FIL_WALLET_ADDRESS", name = "FIL ADDRESS")]
     pub wallet_address: String,
 
+    /// Unique identifier of the Filecoin Station (required).
+    #[arg(long, short = 's', env = "STATION_ID", name = "STATION ID")]
```
Comment (bajtos) [inline daemon/args.rs:13]: I feel the short-hand name `-s` is confusing, e.g. `--state_root` can be abbreviated as `-s` too.

Let's not define any short-hand name for this arg.

```suggestion
    #[arg(long, env = "STATION_ID", name = "STATION ID")]
```

**Review by bajtos — CHANGES_REQUESTED** [review]
Nice! The changes look great. 👏🏻 

I'd like to change one minor detail; the rest LGTM. 👍🏻 

Also let's wait for the CI results to ensure all is green.

**Comment by bajtos** [comment]
https://github.com/filecoin-station/zinnia/actions/runs/8692944647/job/23914200368?pr=520

```
Error: Unknown release type "Feat" found in pull request title "Feat: Receive station_id from Core and push to module as an env var". 

Available types:
 - feat
 - fix
 - chore
 - docs
 - deps
 - test
 - ci
 - refactor
```

To speed up the process, I changed the PR from "draft" to a regular one, update the PR title and will proceed to land & publish this change.

**Review by bajtos — APPROVED** [review]

**Pull request merged**